server/views/components/image-gallery/index.njk

```diff
@@ -1,19 +1,10 @@
 <div class="image-gallery">
   {% for item in imageGallery.items %}
-    <div class="image-gallery__item">
-      <figure class="figure figure--max-out-height image-gallery__figure">
-        <picture class="figure__picture figure__image-container">
-          <img class="figure__image image-gallery__image" src="{{ item.contentUrl }}" alt="{{ item.caption }}" />
-          <source media="(min-width: {{ item.width }}px)" srcset="{{ item.contentUrl }}" />
-        </picture>
-
-        <figcaption class="figure__caption figure__caption--no-underline image-gallery__figure-caption">
-          <div class="flex">
-            <div>{% icon 'other/picture' %}</div>
-            <div>{{ item.caption }}</div>
-          </div>
-        </figcaption>
-      </figure>
-    </div>
+  {% set smallWidth = item.width/item.height*204 %}
```

Comment:
It may not be the same. These calculations are related to the image-gallery.scss file, so should only apply to them.